server/db/models/Gig.js

```diff
@@ -10,6 +10,49 @@ module.exports = function (sequelize, DataTypes) {
   }, {
     classMethods: {
 
+      getGigs: function (organizationId) {
+        return Gig.findAll({
+          where: {
+            organizationId: organizationId
+          }
+        });
+      },
+
+      createGig: function (gigParams) {
+        return Gig.create(gigParams);
+      },
+
+      getGigInfo: function (id) {
+        return Gig.find({
+          where: {
+            id: id
+          }
+        });
+      },
+
+      updateGigInfo: function (id, params) {
+        return Gig.update(params, {
+          where: {
+            id: id
+          }
+        });
+      },
+
+      deleteGig: function (id) {
+        return Gig.find({
```

Comment:
Awesome just tried that out and it totally works, good catch! Im going to return a 204 to indicate resource has been modified (even if it didnt exist in the db)